docs/notebooks/tutorial.ipynb

```diff
@@ -44,30 +44,20 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "metadata": {
     "collapsed": false
    },
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "    # Load price data from yahoo.\r\n",
-      "    data = load_from_yahoo(stocks=['AAPL'], indexes={}, start=start,\r\n",
-      "                           end=end)\r\n",
-      "\r\n",
-      "    # Create and run the algorithm.\r\n",
-      "    algo = TradingAlgorithm(initialize=initialize, handle_data=handle_data,\r\n",
-      "                            identifiers=['AAPL'])\r\n",
-      "    results = algo.run(data)\r\n",
-      "\r\n",
-      "    analyze(results=results)\r\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "!tail ../../zipline/examples/buyapple.py"
+    "# assuming you're running this notebook in zipline/docs/notebooks\n",
+    "import sys\n",
+    "\n",
+    "if 'win' == sys.platform[:3]:\n",
```

Comment:
I think I had used `sys.platform` before in some other work. I wasn't sure why `os.name` returned `nt` for Windows until now ( ref -> https://en.wikipedia.org/wiki/Windows_NT)